axum/src/docs/middleware.md

```diff
@@ -170,18 +170,18 @@ mentally which is one of the reasons `ServiceBuilder` is recommended.
 axum offers many ways of writing middleware, at different levels of abstraction
 and with different pros and cons.
 
-## `axum::middleware::from_fn`
+## `middleware::from_fn`
```

Comment:
I think we should keep the `axum::` prefix here since this section contains references to tower and users might not know if `middleware::from_fn` exists in axum or tower.

Reply:
Reasoning behind this was, most of the links referencing to other crates already had crate name in it like `tower::Layer/Service` rather than just `Layer/Serive`. Making `axum` kind of redundant and `axum::middleware::*` really long for a heading.

Reply:
I still think we should just keep it like it was 🤷